project/Dependencies.scala

```diff
@@ -13,8 +13,9 @@
 import sbt._
 
 object Dependencies {
-  val maxmind   = "com.maxmind.geoip2" %  "geoip2"          % "2.11.0"
-  val collUtils = "com.twitter"        %% "util-collection" % "18.2.0"
-  val scalaz    = "org.scalaz"         %% "scalaz-core"     % "7.0.9"
-  val specs2    = "org.specs2"         %% "specs2-core"     % "4.0.3"  % "test"
+  val maxmind    = "com.maxmind.geoip2" %  "geoip2"          % "2.11.0"
+  val collUtils  = "com.twitter"        %% "util-collection" % "18.2.0"
+  val scalaz     = "org.scalaz"         %% "scalaz-core"     % "7.0.9"
+  val catsEffect = "org.typelevel"      %% "cats-effect"     % "1.0.0-RC2"
```

Comment:
I'm not sure 1.0.0 will be released in time

Reply:
as I feared it's planned for early July: https://gitter.im/typelevel/cats-effect?at=5b19188f160c567d160e4853

Reply:
Sorry, copy and  pasted from version from the documentation without thinking. I'll change it to `0.10.1`.